src/main/scala/uk/gov/nationalarchives/consignmentexport/BagMetadata.scala

```diff
@@ -28,7 +35,7 @@ class BagMetadata(keycloakClient: KeycloakClient)(implicit val logger: SelfAware
     val consignmentType: Option[String] = consignment.consignmentType
     val startDatetime: Option[String] = consignment.createdDatetime.map(_.toFormattedPrecisionString)
     val completedDatetime: Option[String] = consignment.transferInitiatedDatetime.map(_.toFormattedPrecisionString)
-    val contactName: String = getContactName(consignment.userid)
+    val userDetails: UserDetails = keycloakClient.getUserDetails(consignment.userid.toString).getUserDetails()
```

Comment:
We have two `getUserDetails` methods now; I was thinking, maybe the keycloak one could be renamed to `getUserRepresentation`?

Reply:
Did a little bit of renaming to better represent what the functions do. Thanks!